src/utils/contract_display.py

```
@@ -0,0 +1,115 @@
+
+def contract_display_name(
+    symbol: str | None,
+    sec_type: str | None,
+    *,
+    right: str | None = None,
+    strike: float | None = None,
+    contract_expiry: str | None = None,
+    contract_month: str | None = None,
+    exchange: str | None = None,
+    trading_class: str | None = None,
+    last_trade_date: str | None = None,
```

Ruff (0.15.2)

[warning] Unused function argument: `last_trade_date`

(ARG001)